definitions/private-law/json/CaseEvent.json

```diff
@@ -1248,6 +1248,52 @@
     "ShowEventNotes": "N",
     "EventEnablingCondition": "caseTypeOfApplication=\"NEVER_SHOW\""
   },
+  {
+    "LiveFrom": "31/03/2022",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "draftAnOrder",
+    "Name": "Draft an order",
+    "Description": "Draft an order",
+    "DisplayOrder": 161,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "ALL_FINAL_ORDERS_ISSUED(doesOrderClosesCase=\"Yes\"):1;*",
```

Comment:
not sure on the post condition. also precondition, guessing started when the case is submitted.